feathr_project/feathr/client.py

```diff
@@ -67,6 +67,7 @@ def __init__(self, config_path:str = "./feathr_config.yaml", local_workspace_dir
         self.logger = logging.getLogger(__name__)
         # Redis key separator
         self._KEY_SEPARATOR = ':'
+        self._COMPOSITE_KEY_EPARATOR = '#'
```

Comment:
Is `EPARATOR` a typo for `SEPARATOR`?

Reply:
Yes, that's a typo. Modified it in the new commit.